internal/disk/luks.go

```diff
@@ -59,3 +64,9 @@ func (lc *LUKSContainer) Clone() Entity {
 		Payload: lc.Payload,
 	}
 }
+
+func (lc *LUKSContainer) SetUUID(rng *rand.Rand) {
```

Comment:
Should this be `GenUUID()`?

Reply:
it should! I think I pushed a fix for this...

Reply:
Should have been fixed in commit 77aa4381e.

Reply:
Ah the fixup slipped into the wrong commit, I will fix.